src/app/modules/game/components/tic-tac-toe/tic-tac-toe.component.html

```diff
@@ -2,13 +2,13 @@
   <!-- game header -->
   <div class="header mb-4">
     <div class="score">
-      <p>Player X: {{ score['X'] }}</p>
-      <p *ngIf="currentPlayer === 'X'" class="active"></p>
+      <p>Player X: {{ score[gameSymbols.X] }}</p>
+      <p *ngIf="currentPlayer === gameSymbols.X" class="active"></p>
```

Comment:
zasto ovde nemamo jedan _div_ a i na njemu _ngClass_ ? @CRYPPTON

Reply:
mislis na klasu 'active'?

Reply:
donekle, deo sa _ngClass_-om treba da pokrije klasu _acitve_ dok ovo sa _div_-om treba da zameni elemnt _p_ jer se on koristi samo za pisanje tekstualnog sadrzaja, a ti nemas ovde to...

Reply:
a nema smisla koristiti _ngIf_ samo za prikaz klase